examples/vector_databases/Using_vector_databases_for_embeddings_search.ipynb

```diff
@@ -0,0 +1,1419 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cb1537e6",
+   "metadata": {},
+   "source": [
+    "# Using Vector Databases for Embeddings Search\n",
+    "\n",
+    "This notebook takes you through a simple flow to download some data, embed it, and then index and search it using a selection of vector databases. This is a common requirement for customers who want to store and search our embeddings with their own data in a secure environment to support production use cases such as chatbots, topic modelling and more.\n",
+    "\n",
+    "### What is a Vector Database\n",
+    "\n",
+    "A vector database is a database made to store, manage and search embedding vectors. The use of embeddings to encode unstructured data (text, audio, video and more) as vectors for consumption by machine-learning models has exploded in recent years, due to the increasing effectiveness of AI in solving use cases involving natural language, image recognition and other unstructured forms of data. Vector databases have emerged as an effective solution for enterprises to deliver and scale these use cases.\n",
+    "\n",
+    "### Why use a Vector Database\n",
+    "\n",
+    "Vector databases enable enterprises to take many of the embeddings use cases we've shared in this repo (question and answering, chatbot and recommendation services, for example), and make use of them in a secure, scalable environment. Many of our customers make embeddings solve their problems at small scale but performance and security hold them back from going into production - we see vector databases as a key component in solving that, and in this guide we'll walk through the basics of embedding text data, storing it in a vector database and using it for semantic search.\n",
+    "\n",
+    "\n",
+    "### Demo Flow\n",
+    "The demo flow is:\n",
+    "- **Setup**: Import packages and set any required variables\n",
+    "- **Load data**: Load a dataset and embed it using OpenAI embeddings\n",
+    "- **Pinecone**\n",
+    "    - *Setup*: Here we setup the Python client for Pinecone. For more details go [here](https://docs.pinecone.io/docs/quickstart)\n",
+    "    - *Index Data*: We'll create an index with namespaces for __titles__ and __content__\n",
+    "    - *Search Data*: We'll test out both namespaces with search queries to confirm it works\n",
+    "- **Weaviate**\n",
+    "    - *Setup*: Here we setup the Python client for Weaviate. For more details go [here](https://weaviate.io/developers/weaviate/current/client-libraries/python.html)\n",
```

Comment:
"setup" is the noun, "set up" is the verb. These should say "set up".

Reply:
Updated